doc/index.rst

```diff
@@ -38,6 +38,36 @@ Some key features include:
     * An API for addition of new cell types, e.g. LIF and Poisson spike generators.
     * Validation tests against numeric/analytic models and NEURON.
 
+Citing Arbor
+------------
+
+.. |DOI-v0.1| image:: https://zenodo.org/badge/DOI/10.5281/zenodo.2583709.svg
+    :target: https://doi.org/10.5281/zenodo.2583709
+
+.. |DOI-v0.2| image:: https://zenodo.org/badge/DOI/10.5281/zenodo.1459679.svg
+     :target: https://doi.org/10.5281/zenodo.1459679
+
```

Comment:
The DOIs and links are the wrong way around: first one is the v0.2 link, the second, the v0.1 link.